src/addrman.cpp

```diff
@@ -29,19 +29,19 @@ static constexpr uint32_t ADDRMAN_NEW_BUCKETS_PER_SOURCE_GROUP{64};
 /** Maximum number of times an address can occur in the new table */
 static constexpr int32_t ADDRMAN_NEW_BUCKETS_PER_ADDRESS{8};
 /** How old addresses can maximally be */
-static constexpr int64_t ADDRMAN_HORIZON_DAYS{30};
+static constexpr auto ADDRMAN_HORIZON{30 * 24h};
 /** After how many failed attempts we give up on a new node */
 static constexpr int32_t ADDRMAN_RETRIES{3};
 /** How many successive failures are allowed ... */
 static constexpr int32_t ADDRMAN_MAX_FAILURES{10};
-/** ... in at least this many days */
-static constexpr int64_t ADDRMAN_MIN_FAIL_DAYS{7};
+/** ... in at least this duration */
+static constexpr auto ADDRMAN_MIN_FAIL{7 * 24h};
```

Comment:
perhaps it's a good opportunity to give a better variable name here.
`ADDRMAN_FAIL_DUR_UNIT` or something

Reply:
Sure, happy to do if people agree. Please leave a comment here or upvote glebs comment.